### PR TITLE
docs: Install instructions in README

### PR DESCRIPTION
**Pull request by fabriziodemaria**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,24 @@ To learn more about the basic concepts (flags, targeting key, evaluation context
 the [OpenFeature reference documentation](https://openfeature.dev/docs/reference/intro) can be
 useful.
 
-# Using the provider
+## Install
+
+### Maven
+
```
Comment (nicklasl) [inline README.md:16]:
```suggestion
 <!---x-release-please-start-version-->
 
```

File: README.md [hunk 2]
```diff
@@ -10,13 +10,24 @@ To learn more about the basic concepts (flags, targeting key, evaluation context
...
+ 
+```xml
+<dependency>
+    <groupId>com.spotify.confidence</groupId>
+    <artifactId>openfeature-provider</artifactId>
+    <version>0.0.3</version>
+</dependency>
+```
+(Make sure to use the latest [release version](https://github.com/spotify/confidence-openfeature-provider-java/releases) when importing the library).
+
```
Comment (nicklasl) [inline README.md:25]:
```suggestion
<!---x-release-please-end-->

```

**Comment by nicklasl** [comment]
If we add the ` <!---x-release-please-start-version-->` and `end` given this line: https://github.com/spotify/confidence-openfeature-provider-java/blob/readme-install/release-please-config.json#L14. Release please will also auto update the version.

**Comment by fabriziodemaria** [comment]
> If we add the   and end given this line: https://github.com/spotify/confidence-openfeature-provider-java/blob/readme-install/release-please-config.json#L14. Release please will also auto update the version.

I didn't know! Cool 👌

**Review by nicklasl — APPROVED** [review]

**Pull request merged**